include/zephyr/sys/util_macro.h

```diff
@@ -43,10 +43,16 @@ extern "C" {
  */
 #define BIT(n)  (1UL << (n))
 #endif
-#endif
+#endif /* BIT */
 
+#ifndef BIT64
```

Comment:
why is extra ifdeffery needed?

Reply:
So that it is same as `BIT()`, but this commit doesn't seem necessary, will drop it.

Reply:
I guess it exists for BIT due to some HALs polluting the namespace. However, the version without `ULL` prefix is likely needed for asm?

Reply:
> I guess it exists for BIT due to some HALs polluting the namespace. However, the version without `ULL` prefix is likely needed for asm?

Had that impression as well, but my application continue to build without the `ULL` prefix